fire/console/console_io.py

```diff
@@ -83,7 +83,11 @@ def More(contents, out, prompt=None, check_pager=True):
     return
   if check_pager:
     pager = encoding.GetEncodedValue(os.environ, 'PAGER', None)
-    if pager == '-':
+    fire_pager = encoding.GetEncodedValue(os.environ, 'FIRE_PAGER', None)
+    if fire_pager:
+      # FIRE_PAGER takes precedence to PAGER
+      pager = fire_pager
+    elif pager == '-':
```

Comment:
How about allowing `FIRE_PAGER=-`, too, so it works consistently like `PAGER`?

```suggestion
    if pager == '-':
```

Reply:
Ope, good catch, we definitely want this. I didn't comprehend this properly. Thanks!